src/main/java/com/gotocompany/firehose/evaluator/CELPayloadEvaluator.java

```diff
@@ -0,0 +1,46 @@
+package com.gotocompany.firehose.evaluator;
+
+import com.google.protobuf.Descriptors;
+import com.google.protobuf.DynamicMessage;
+import com.google.protobuf.Message;
+import lombok.extern.slf4j.Slf4j;
+import org.projectnessie.cel.checker.Decls;
+import org.projectnessie.cel.tools.Script;
+import org.projectnessie.cel.tools.ScriptCreateException;
+import org.projectnessie.cel.tools.ScriptException;
+import org.projectnessie.cel.tools.ScriptHost;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
+public class CELPayloadEvaluator implements PayloadEvaluator<Message> {
```

Comment:
Since schema could be updated on runtime, should we also refresh this object (similar to how stencil library works)